frontend/templates/troubleshooting/index.tsx

```diff
@@ -46,6 +57,51 @@ const Wiki: NextPageWithLayout<{
    );
 };
 
+function NavBar() {
+   return (
+      <Flex
+         w="100%"
+         h="48px"
+         backgroundColor="white"
+         borderBottomColor="gray.200"
+         borderBottomWidth="1px"
+         justify="center"
+      >
+         <Flex maxW="1280px" flexGrow="1">
+            <Box flexGrow="1">Breadcrumbs</Box>
+            <Box>Parts/Guide/Answers</Box>
+            <EditButton />
+         </Flex>
+      </Flex>
+   );
+}
+
+function EditButton() {
+   return (
+      <Button
+         leftIcon={<FaIcon icon={faPenToSquare} />}
+         variant="link"
+         bgColor="transparent"
+         textColor="brand"
+         borderLeftColor="gray.200"
+         borderLeftWidth="1px"
+         borderRadius="0px"
+         padding="9px, 16px, 9px, 16px"
+      >
+         <Text
+            fontFamily="SF Pro"
```

Comment:
Do we want to specify a font family? I am completely out of the loop about our usage of fonts, but I thought we were going to use system fonts?

Reply:
I think this got fixed in a later commit.